plugins/permission-react/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@backstage/plugin-permission-react",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "homepage": "https://backstage.io",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/backstage",
+    "directory": "plugins/permission"
+  },
+  "keywords": [
+    "backstage"
+  ],
+  "scripts": {
+    "build": "backstage-cli plugin:build",
```

Comment:
I think it would be a slightly better fit to skip the `start` command and to make this use the plain `backstage-cli build` command instead, since this is more of a library package